components/process.rst

```diff
@@ -352,6 +352,35 @@ The input of a process can also be defined using `PHP streams`_::
     // will echo: 'foobar'
     echo $process->getOutput();
 
+Using TTY and PTY Modes
+-----------------------
+
+All examples above show that your program has control over the input of a
+process (using ``setInput()``) and the output from that process (using
+``getOutput()``). The Process component has two special modes that tweak
+the relationship between your program and the process: teletype (tty) and
```

Comment:
Maybe I'm *nitpicking*, but this is slightly confusing to me:

```
the relationship between your program and the process
```

"your program" is the Symfony application ... or the code of my Symfony app which is using Symfony's Process class?

"the process" is the code using Symfony's Process class ... or is it the process being executed by the operating system?

Reply:
Hmm, I'm not sure what you mean. As tty replaces input/output with the input/output of the process, it can only be run as a command (running it in e.g. a controller produces an error). So "your program" is the code of the Symfony app which uses the Process class